_posts/2023-09-18-the-definitive-guide-2-cn.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Chinese edition of Trino: The Definitive Guide"
+author: Manfred Moser, Martin Traverso, Matt Fuller
+image: /assets/ttdg2-cn-cover.png
+excerpt_separator: <!--more-->
+show_pagenav: false
+---
+
+Trino, Trino, Trino everywhere. Just looking at our website stats and the users
+in our community chat, we know that Trino is going places. We also know that one
+of these places with a large user community is China. And now we have good
+news for you.
```

Comment:
Let's mention the good news in the opening paragraph/preview? I don't think we want to clickbait people into our posts or bury the lede like this.

Reply:
Well... the title already says it .. but I can repeat if you think this is necessary.

Reply:
Done